glib/src/auto/functions.rs

```diff
@@ -97,6 +97,21 @@ pub fn compute_checksum_for_data(
     }
 }
 
+#[doc(alias = "g_compute_checksum_for_string")]
+pub fn compute_checksum_for_string(
+    checksum_type: ChecksumType,
+    str: &str,
```

Comment:
This should probably become

```suggestion
    str: impl IntoGStr,
```

to avoid copies/allocations if we can

Reply:
This is in  the auto/ which is wrong btw

Reply:
Thanks, fixed

Reply:
No, the fix is wrong. See other functions in glib/src/functions. That is where you should add the function.

Reply:
Could also add it to `src/checksum.rs` as an associated function to `Checksum`. Either works IMHO

Reply:
> No, the fix is wrong. See other functions in glib/src/functions. That is where you should add the function.

I'm unable to figure out where to add the function - the other functions don't seem to be in alphabetical order...